### ISSUE TITLE
[BUG] Stacksize when generating the code (problem calculating the entity weight)

### ISSUE THREAD

**Issue by ibuilder** [TiagoSilvaPereira/vemto-issues#13]
```
RangeError: Maximum call stack size exceeded
    at d.belongsTo (app://./js/chunk-common.9dc99885.js:1:43385)
    at app://./js/generator.e6e5039d.js:1:32354
    at Array.forEach (<anonymous>)
    at Ie.calculateEntityWeight (app://./js/generator.e6e5039d.js:1:32323)
    at app://./js/generator.e6e5039d.js:1:32382
    at Array.forEach (<anonymous>)
    at Ie.calculateEntityWeight (app://./js/generator.e6e5039d.js:1:32323)
    at app://./js/generator.e6e5039d.js:1:32382
    at Array.forEach (<anonymous>)
    at Ie.calculateEntityWeight (app://./js/generator.e6e5039d.js:1:32323)
```

---

vemto 0.4.15
Electron 8.5.0
win32 10.0.19041
Locale: en-US

**Comment by TiagoSilvaPereira** [comment]
Hi @ibuilder thanks for reporting. Could you please add a screenshot of the schema screen so I can reproduce the issue?

**Comment by TiagoSilvaPereira** [comment]
Fixed on 0.5.2
